### DISCUSSION TITLE
v1.0.5

### DISCUSSION THREAD

**Discussion by DoneDeal0** [DoneDeal0/superdiff#5] [Announcements]
- `getObjectDiff()` and  `isEqual` now accept a facultative `options` parameter:

```ts
{
  discardArrayOrder?: boolean // false by default
}
```
If `discardArrayOrder` is set to true, `["hello", "world"]` and `["world", "hello"]` will be considered as `equal`, because the two arrays have the same values, just not in the same order.

- add buy me a coffee link. If you use the lib, feel free to show your support.

<hr /><em>This discussion was created from the release <a href='https://github.com/DoneDeal0/superdiff/releases/tag/v1.0.5'>v1.0.5</a>.</em>

**Comment by candidia** [comment]
discard vs disregard?

**Reply by DoneDeal0** [reply]
Hi @CSFlorin !`disregard` is definitely better than `discard`. English is not my native language, I make some stupid mistakes sometimes^^. I'll correct it in a future version. Thanks for pointing it out!

**Reply by candidia** [reply]
I see "ignore" most often by the way

**Reply by DoneDeal0** [reply]
`discardArrayOrder` is now renamed as `ignoreArrayOrder` (see v1.0.9 [release note](https://github.com/DoneDeal0/superdiff/releases/tag/v1.0.9)). Thanks again for your suggestion.
